atmos_shared/atmos_tracer_driver.F90

```diff
@@ -19,6 +19,7 @@
 !***********************************************************************
 
 module atmos_tracer_driver_mod
+#include <fms_platform.h>
```

Comment:
I don't think we need both fms_platform.h and the use_module platform_mod.

Reply:
Good catch, removed.